### PR TITLE
[MNOE-399] Audit log in frontend

### PR DESCRIPTION
**Pull request by ouranos**
Blocked by https://github.com/maestrano/mno-enterprise/pull/272 in mnoe for backend + locales

image

### CODE REVIEW

File: src/app/components/mno-audit-events-list/mno-audit-event-list.less [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      background-color: @table-cell-row-even-bg-color
+    }
+
+    &:nth-of-type(odd) {
+      background-color: @table-cell-row-odd-bg-color
+    }
+  }
+}
+
+
```
Comment (alexnoox) [inline src/app/components/mno-audit-events-list/mno-audit-event-list.less]: Too many empty lines

File: src/app/views/company/audit-log/organization-audit-log.directive.coffee [hunk 2]
```diff
@@ -0,0 +1,19 @@
+DashboardOrganizationAuditLogCtrl = ($scope, MnoeAuditEvents) ->
+  'ngInject'
+
+  #====================================
+  # Pre-Initialization
+  #====================================
+  $scope.isLoading = true
+  $scope.exportUrl = MnoeAuditEvents.exportUrl()
```
Comment (alexnoox) [inline src/app/views/company/audit-log/organization-audit-log.directive.coffee]: Are you sure this controller doesn't need a `return` instruction at the end?

File: src/app/views/company/company.html [hunk 3]
```diff
@@ -128,6 +128,23 @@ <h2 class="section-title">{{ 'mno_enterprise.templates.dashboard.organization.in
...
+      <!-------------------------------------->
+      <div uib-tab active="vm.tabs.audit_log">
+        <div uib-tab-heading>{{ 'mno_enterprise.templates.dashboard.organization.index.audit_log.title' | translate }}</div>
+
+        <!-- Directive: DashboardOrganizationAuditLog -->
+        <div class="well">
+          <div class="col-md-12">
+            <div dashboard-organization-audit-log></div>
+          </div>
+          <div class="clearfix"></div>
```
Comment (alexnoox) [inline src/app/views/company/company.html]: Not sure about the empty `div` with a `clearfix`...

File: src/app/views/company/audit-log/organization-audit-log.directive.coffee (hunk 2 above)
Comment (ouranos) [inline src/app/views/company/audit-log/organization-audit-log.directive.coffee]: I've based it on other directives that don't have a return

**Pull request merged**